docs/user-guide/host-settings.md

```diff
@@ -0,0 +1,194 @@
+attaching to the Open Network Fabric. The steps to configure the bond are the
+same as the [above server settings](#server-settigns)
+
+The kernel bonding driver discusses other modes for bonding, several of those modes don't require cooperation on the
+switch side, so there is no fabric configuration needed. The primary concern
+for using these modes is out of order TCP delivery and fault detection and
+correction time.
+
+## Active / Passive Fault Tolerance
+
+Some networking equipment doesn't support MCLAG or ESLAG. If a user would like
+the host to have a fault tolerant availability the Linux server bond can be
+configured for active / standby. In this topology, a link is connected from
+server-1 to switch 1 and another link is connected from server-1 to switch 2.
+There is no [MCLAG connection](connections.md#mclag) nor [switch
+group](devices.md#redundancy-groups) configuration applied to the
+switches. When adding these links to the fabric via the `kubectl` command or
+the wiring diagram, use the [unbundled](connections.md#unbundled) connection
+kind.
+
+### Active / Passive Bond Settings
+
+On the host:
+
+1. create a bond: 
+``` bash
+sudo ip link add bond0 type bond mode active-backup miimon 50 num_grat_arp 5 primary enp2s1
+```
+2. add the slaves to the bond:
+``` bash
+  sudo ip link set link enp2s1 master bond0
+  sudo ip link set link enp2s2 master bond0
+```
+3. Add an address to the bond:
+``` bash
+  sudo ip addr add 10.30.10.10/24 dev bond0`
+```
+4. Make the bond  the default route:
+```bash
+  sudo ip route add default via 10.30.10.1
+```
+
+View the link layer information for the interfaces to see that the same MAC
+address is on all 3 interfaces: bond0, enp2s1,enp2s2:
+``` console 
+core@server-1 ~ $ ip link
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+2: enp2s1: <BROADCAST,MULTICAST,SLAVE,UP,LOWER_UP> mtu 1500 qdisc mq master bond0 state UP mode DEFAULT group default qlen 1000
+    link/ether a6:eb:64:de:95:92 brd ff:ff:ff:ff:ff:ff permaddr a0:36:9f:3f:1a:e8
+3: enp2s2: <BROADCAST,MULTICAST,SLAVE,UP,LOWER_UP> mtu 1500 qdisc mq master bond0 state UP mode DEFAULT group default qlen 1000
+    link/ether a6:eb:64:de:95:92 brd ff:ff:ff:ff:ff:ff permaddr 14:02:ec:7f:3a:3c
+4: enp2s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP mode DEFAULT group default qlen 1000
+    link/ether 0c:20:12:fe:01:00 brd ff:ff:ff:ff:ff:ff
+10: bond0: <BROADCAST,MULTICAST,MASTER,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default qlen 1000
+    link/ether a6:eb:64:de:95:92 brd ff:ff:ff:ff:ff:ff
+13: bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default qlen 1000
+    link/ether a6:eb:64:de:95:92 brd ff:ff:ff:ff:ff:ff
+```
+
+Confirm that the bonding driver is reporting the selected primary interface:
+```console
+cat /proc/net/bonding/bond0
```

Comment:
```suggestion
core@server-1 ~ $ cat /proc/net/bonding/bond0
```

With `console`, you need a prompt so the highlighter can make the distinction between commands and output.